web/src/views/SubmissionPortal/Components/HarmonizerSidebar.vue

```diff
@@ -0,0 +1,162 @@
+<script lang="ts">
+import {
+  defineComponent, PropType, ref,
+} from '@vue/composition-api';
+import FindReplace from '@/views/SubmissionPortal/Components/FindReplace.vue';
+import type { HarmonizerApi } from '@/views/SubmissionPortal/harmonizerApi';
+import ContactCard from '@/views/SubmissionPortal/Components/ContactCard.vue';
+import ImportExportButtons from '@/views/SubmissionPortal/Components/ImportExportButtons.vue';
+import ColumnHelp from '@/views/SubmissionPortal/Components/ColumnHelp.vue';
+import MetadataSuggester from '@/views/SubmissionPortal/Components/MetadataSuggester.vue';
+import { ColumnHelpInfo, HarmonizerTemplateInfo } from '@/views/SubmissionPortal/types';
+
+/**
+ * The tabbed Data Harmonizer sidebar.
+ */
+export default defineComponent({
+  components: {
+    MetadataSuggester,
+    ColumnHelp,
+    ImportExportButtons,
+    ContactCard,
+    FindReplace,
+  },
+  props: {
+    /**
+     * Help information for the currently selected column.
+     */
+    columnHelp: {
+      type: Object as PropType<ColumnHelpInfo | null>,
+      default: null,
+    },
+    /**
+     * Information about the active template.
+     */
+    harmonizerTemplate: {
+      type: Object as PropType<HarmonizerTemplateInfo>,
+      required: true,
+    },
+    /**
+     * Whether the current user is allowed to edit metadata.
+     */
+    metadataEditingAllowed: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * The Harmonizer API instance.
+     */
+    harmonizerApi: {
+      type: Object as PropType<HarmonizerApi>,
+      required: true,
+    },
+  },
+  emits: ['export-xlsx', 'import-xlsx'],
+  setup(props, { emit }) {
+    const tabModel = ref(0);
+    const TABS = [
+      {
+        icon: 'mdi-information-outline',
+        label: 'Column Help',
```

Comment:
```suggestion
        icon: 'mdi-information-outline',
        label: 'Column Info',
```
I think having "Column Help" and "Help" could potentially be confusing. This label matches the icon better and I think still fits what this column has to offer.

Reply:
I updated the tooltip and title within the sidebar to "Column Info".